### PR TITLE
Switch beta to head of current dev (kodi 20)

### PR DESCRIPTION
**Pull request by razzeee**

### CODE REVIEW

File: addons/inputstream.adaptive/inputstream.adaptive.json [hunk 1]
```diff
@@ -1,12 +1,91 @@
...
-            "commit": "58c54c7b641f69a862ed2fa5a640b5fa2db9c711"
+  "name": "inputstream.adaptive",
+  "buildsystem": "cmake-ninja",
+  "config-opts": ["-DBUILD_TESTING=OFF"],
+  "sources": [
+    {
+      "type": "git",
+      "url": "https://github.com/peak3d/inputstream.adaptive",
+      "tag": "20.0.3-Nexus",
+      "commit": "0b2715a282f3726d089fae592c46efd425aa5b49"
```
Comment (wsnipex) [inline addons/inputstream.adaptive/inputstream.adaptive.json:10]:
```suggestion
      "commit": "785d2b10542f84417189409044b1165d47df23e2"
```

File: addons/inputstream.adaptive/inputstream.adaptive.json [hunk 2]
```diff
@@ -1,12 +1,91 @@
...
-            "url": "https://github.com/peak3d/inputstream.adaptive",
-            "tag": "2.6.7-Matrix",
-            "commit": "58c54c7b641f69a862ed2fa5a640b5fa2db9c711"
+  "name": "inputstream.adaptive",
+  "buildsystem": "cmake-ninja",
+  "config-opts": ["-DBUILD_TESTING=OFF"],
+  "sources": [
+    {
+      "type": "git",
+      "url": "https://github.com/peak3d/inputstream.adaptive",
```
Comment (wsnipex) [inline addons/inputstream.adaptive/inputstream.adaptive.json:8]:
```suggestion
      "url": "https://github.com/xbmc/inputstream.adaptive",
```

File: addons/inputstream.adaptive/inputstream.adaptive.json [hunk 3]
```diff
@@ -1,12 +1,91 @@
...
-    "buildsystem": "cmake-ninja",
-    "sources": [
-        {
-            "type": "git",
-            "url": "https://github.com/peak3d/inputstream.adaptive",
-            "tag": "2.6.7-Matrix",
-            "commit": "58c54c7b641f69a862ed2fa5a640b5fa2db9c711"
+  "name": "inputstream.adaptive",
+  "buildsystem": "cmake-ninja",
+  "config-opts": ["-DBUILD_TESTING=OFF"],
```
Comment (wsnipex) [inline addons/inputstream.adaptive/inputstream.adaptive.json:4]:
```suggestion
  "config-opts": ["-DBUILD_TESTING=OFF", "-DENABLE_INTERNAL_BENTO4=ON"],
```

File: addons/inputstream.adaptive/inputstream.adaptive.json [hunk 4]
```diff
@@ -1,12 +1,91 @@
...
+  "sources": [
+    {
+      "type": "git",
+      "url": "https://github.com/peak3d/inputstream.adaptive",
+      "tag": "20.0.3-Nexus",
+      "commit": "0b2715a282f3726d089fae592c46efd425aa5b49"
+    }
+  ],
+  "modules": [
```
Comment (wsnipex) [inline addons/inputstream.adaptive/inputstream.adaptive.json:13]: bento4 can be removed here, including all the patches

**Review by wsnipex — CHANGES_REQUESTED** [review]
let's try to get inputstream.adaptive fixed.

File: addons/inputstream.adaptive/inputstream.adaptive.json [hunk 5]
```diff
@@ -1,12 +1,15 @@
...
+    ],
     "sources": [
         {
             "type": "git",
-            "url": "https://github.com/peak3d/inputstream.adaptive",
-            "tag": "2.6.7-Matrix",
-            "commit": "58c54c7b641f69a862ed2fa5a640b5fa2db9c711"
+            "url": "https://github.com/xbmc/inputstream.adaptive",
+            "commit": "785d2b10542f84417189409044b1165d47df23e2"
         }
```
Comment (wsnipex) [inline addons/inputstream.adaptive/inputstream.adaptive.json:13]:
```suggestion
        },
        {
              "type": "archive",
              "url": "https://github.com/axiomatic-systems/Bento4/archive/refs/tags/v1.6.0-639.tar.gz",
              "sha256": "9f3eb912207d7ed9c1e6e05315083404b32a11f8aacd604a9b2bdcb10bf79eb9",
              "dest": "bento4_src",
              "dest-filename:" bento4.tar.gz"
```

File: addons/inputstream.adaptive/inputstream.adaptive.json [hunk 6]
```diff
@@ -1,12 +1,15 @@
 {
     "name": "inputstream.adaptive",
     "buildsystem": "cmake-ninja",
+    "config-opts": [
+        "-DBUILD_TESTING=OFF",
+        "-DENABLE_INTERNAL_BENTO4=ON"
```
Comment (wsnipex) [inline addons/inputstream.adaptive/inputstream.adaptive.json:6]:
```suggestion
        "-DENABLE_INTERNAL_BENTO4=ON",
        "-DBENTO4_URL=bento4_src"
```

**Review by wsnipex — COMMENTED** [review]
untested: see suggestion for how to hopefully fix IA/bento4.

Otherwise feel free to revert to  what you had before.

**Review by wsnipex — APPROVED** [review]

**Comment by hfiguiere** [comment]
this MR talks about `beta` but was merged on `master` aka `stable`. So now the stable flatpak has a pre-release version

**Comment by wsnipex** [comment]
yep, we're trying to get this resolved

**Pull request merged**